app/inbox/utils.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+"""Define util for the inbox app.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+from inbox.models import Notification
+
+
+def send_notification_to_user(from_user, to_user, cta_url, cta_text, msg_html):
```

Comment:
1. it seems like this is only called in once place.  it needs to be littered throughout the codebase.
2. when it is placed across the codebase.. can we please use django signals to do it?  the current pattern of pasting it in a bunch of views is not sustainable.. it should really like in a django signal to reduce coupling increase logical coherence of the codebase.

Reply:
Django signals are called only when we save a given model. I have a question:
 Are we calling .save() or are we using .update() to make changes to the bounty, interests?